src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -277,6 +285,10 @@ namespace OpenMS
     // TODO make sure that only assembled traces (more than one trace -> has a charge)
     // see FeatureFindingMetabo: defaults_.setValue("remove_single_traces", "false", "Remove unassembled traces (single traces).");
     Size seeds_added(0);
+
+    // WARNING: Superhack! Use unique ID to distinguish seeds from real IDs.
```

Comment:
add note why such a large value is used?

Reply:
yes